components/sbm-core/src/main/java/org/springframework/sbm/build/impl/OpenRewriteMavenBuildFile.java

```diff
@@ -756,8 +766,19 @@ private Plugin mapToPlugin(Xml.Tag tag) {
                 if (versionTag.isPresent()) {
                     version = versionTag.get().getValue().get();
                 }
-                Plugin plugin = new Plugin(groupId, artifactId, version, List.of(), "", "");
-                return plugin;
+				Optional<Xml.Tag> configurationTag = tag.getChild("configuration");
+				HashMap<String, Object> configuration = new HashMap<>();
+				if (configurationTag.isPresent() && configurationTag.get().getChildren().size() > 0){
+					try {
+						String configurationXml = configurationTag.get().print(new XmlPrinter<>());
```

Comment:
Any reason not using `configurationTag.get().getValue()` instead of `print(new XmlPrinter())` here?